src/ElvClient.js

```diff
@@ -694,8 +714,39 @@ class ElvClient {
     };
   }
 
-  WriteTokenNodeUrl({writeToken}) {
-    const nodeUrl = this.HttpClient.draftURIs[writeToken];
+  /**
+   * Return node url for a given write token
+   *
+   * @methodGroup Nodes
+   * @namedParams
+   * @param {string} writeToken - The write token to match to a node
+   * @param {boolean=} networkCall - If specified, will make a network call to /nodes?token=<writeToken>
+   *
+   * @returns {Promise<string>} - The node url for a write token
+   */
+  async WriteTokenNodeUrl({writeToken, networkCall=false}) {
```

Comment:
having networkCall as a param here kind of defeats the purpose of offering local lookup because the method is still async. I recommend creating a separate, non async method for looking up node by saved local info if that's actually used